test/support/mocks/graph-client.service.mock.ts

```diff
@@ -0,0 +1,57 @@
+import { Client, GraphRequest } from '@microsoft/microsoft-graph-client';
+import { when } from 'jest-when';
+
+class MockClient {
+  api: jest.Mock;
+  constructor() {
+    this.api = jest.fn();
+  }
```

Comment:
Can we have a blank line between the fields and the constructor for all the classes in this file please?